pkgs/tools/misc/cyberchef/generate.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env nix-shell
+#! nix-shell -i bash -p nodePackages.node2nix jq gnutar gzip curl moreutils
+set -euo pipefail
+
+VERSION=9.37.3
+
```

Comment:
That will not work with the auto update bot.

Reply:
Yes, unfortunately.

Reply:
Can we make it work though?

Reply:
Ah, I didn't know the bot could run scripts like that.

It should work now, when running locally it produces a reasonable diff and then errors on `fatal: path 'pkgs/tools/misc/cyberchef/default.nix' exists on disk, but not in 'remotes/upstream/master'`